tests/utils.go

```diff
@@ -478,7 +478,7 @@ func WaitForAllPodsReady(timeout time.Duration, listOptions metav1.ListOptions)
 		virtClient, err := kubecli.GetKubevirtClient()
 		PanicOnError(err)
 
-		podsList, err := virtClient.CoreV1().Pods(k8sv1.NamespaceAll).List(listOptions)
+		podsList, err := virtClient.CoreV1().Pods(KubeVirtInstallNamespace).List(listOptions)
```

Comment:
The main issue I see here is that this is tied with a mechanism which allows deploying arbitrary manifests as additional testing infrastructure. These additional manifests are not tied to the kubevirt install namespaces.

Reply:
In this case the check would for instance not work anymore with the default testing mainfests in this repo. We would start with the test too early.

Reply:
I would have to trust you here that we need to watch more than our own namespace, even though I don't understand what else we should be watching.